metpy/calc/basic.py

```diff
@@ -220,3 +224,34 @@ def heat_index(temperature, rh, mask_undefined=True):
             hi = units.Quantity(masked_array(hi, mask=mask), hi.units)
 
     return hi
+
+
+@exporter.export
+def pressure_to_height_std(pressure):
+    '''Convert pressure data to heights using the U.S. standard atmosphere.
```

Comment:
WOW. Change the starting `'''` of the string to `r'''`. This tells Python to treat it as a raw string--which will keep it from treating things like `\f` in the docstring as special characters. The only way I found this was by comparing against the docstring above.